game-core/src/main/java/games/strategy/engine/lobby/server/GameDescription.java

```diff
@@ -134,32 +79,16 @@ public void setGameName(final String gameName) {
     this.gameName = gameName;
   }
 
-  public void setHostedBy(final INode hostedBy) {
-    version++;
-    this.hostedBy = hostedBy;
-  }
-
   public void setPlayerCount(final int playerCount) {
     version++;
     this.playerCount = playerCount;
   }
 
-  @Deprecated
-  public void setPort(final int port) {
-    version++;
-    this.port = port;
-  }
-
   public void setRound(final String round) {
     version++;
     this.round = round;
   }
 
-  public void setStartDateTime(final Instant startDateTime) {
-    version++;
```

Comment:
I keep re-forgetting why this version is here, not sure if I really understand why :disappointed: 

Anyways, are you sure it's safe to remove the `version` increment?

Reply:
This version is a sort-of guarantee that only the last call is being processed last, even if other calls that were made earlier arrive later at the class.
I don't think it's needed because the engine shouldn't really be updating the description so fast that the server can't keep up with it.
Ideally we'd even drop the mutable state and use `@Wither` or something and increment a static id, every time a new instance is created if we want to be sure.

Reply:
Thanks, at the sounds of it, if it's overkill, it seems like it is overkill. Would it make sense to remove the `version` value, or add a TODO that we will be removing it? Or would you iterate more in this code area and take care of it soon? I'm just concerned about eventual confusion that sometimes the value is incremented and others not.